stackify/constants.py

```diff
@@ -4,6 +4,11 @@
 API_URL = 'https://api.stackify.com'
 IDENTIFY_URL = '/Metrics/IdentifyApp'
 LOG_SAVE_URL = '/Log/Save'
+
+DEFAULT_SOCKET_FILE = '%2Fusr%2Flocal%2Fstackify%2Fstackify.sock'
```

Comment:
why %2 instead of / ?

Reply:
thats the ugly thing about the library we are using as of now. it will not work if using `/`

Reply:
please add a comment in the code detailing that.